demo/aevnmt-translate.sh

```diff
@@ -6,7 +6,7 @@ TAG=flickr
 SRC=de
 TGT=en
 DATA=${TAG}/data
-TRAINED_MODEL=${TAG}/models/aevnmt/supervised
+TRAINED_MODEL=${TAG}/models/aevnmt/supervised.${SRC}-${TGT}
```

Comment:
Will this not break existing downloaded models?

Reply:
Compatibility with existing models is long gone (though architecture blocks may remain the same, refactoring affected the parameters' names). The best thing would be to retrain those models (perhaps have different trained models for different versions of the code.

Reply:
I see, I'll have a look at that later then.